Documentation/ceph-disaster-recovery.md

```diff
@@ -81,18 +81,18 @@ ceph-mon \
     --log-stderr-prefix=debug \
     --default-log-to-file=false \
     --default-mon-cluster-log-to-file=false \
-    --mon-host=$(ROOK_CEPH_MON_HOST) \
-    --mon-initial-members=$(ROOK_CEPH_MON_INITIAL_MEMBERS) \
+    --mon-host=$ROOK_CEPH_MON_HOST \
+    --mon-initial-members=$ROOK_CEPH_MON_INITIAL_MEMBERS \
     --id=b \
     --setuser=ceph \
     --setgroup=ceph \
     --foreground \
     --public-addr=10.100.13.242 \
     --setuser-match-path=/var/lib/ceph/mon/ceph-b/store.db \
-    --public-bind-addr=$(ROOK_POD_IP)
+    --public-bind-addr=$ROOK_POD_IP
 ```
 
-(be sure to remove the single quotes around the `--log-stderr-prefix` flag)
+(be sure to remove the single quotes around the `--log-stderr-prefix` flag and the paranthesis around the variables being passed ROOK_CEPH_MON_HOST, ROOK_CEPH_MON_INITIAL_MEMBERS and ROOK_POD_IP )
```

Comment:
Is this line still needed? Looks like this PR already removes the parens.

Reply:
In the yaml generated - the '' and () appears, hence I thought of extending the instruction to remove the parenthesis.

Reply:
Ok, I see now where you're referring to in the yaml that comes from the pod spec, thanks